lib/facility.rb

```diff
@@ -1,14 +1,64 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name,
+              :address, 
+              :phone, 
+              :services,
+              :registered_vehicles,
+              :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(info)
+    @name = info[:name] || info[:dmv_office] || info[:office_name] 
```

Comment:
Yup, this works! You can also abstract this in a method called something like `find_name` that has this logic in it, then line 10 could be something like `@name = find_name(info)`.